app/models/topic.rb

```diff
@@ -0,0 +1,12 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id         :bigint           not null, primary key
+#  name       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+class Topic < ApplicationRecord
+end
```

Comment:
Also i believe that you should add a validate here for the name attribute, the validate will check the constraint before add the record in the databases, so you will save a step